### PR TITLE
Change interaction while asking the flavor

### PR DESCRIPTION
**Pull request by geovanisouza92**
- [x] I ran `npm test` for the package I'm modifying
- [x] I used `npm run commit` instead of `git commit`

From now, the user is asked to choose from existing "core" flavors, with the option to discover
more, and then choosing the stream library

ISSUES CLOSED: #2, #16

### CODE REVIEW

**Comment by nickbalestra** [comment]
@geovanisouza92 I've tried to bring this a bit forward let me know what you think. Shortly:
- splitted the flavor registry over 2 json files: `coreFlavors.json` and `communityFlavors.json`
- Sorted the flavors by name 
- After discovery run, show community flavor sorted by name `+` core flavors `-` `'run-discovery'`

Questions I still have is related to naming: do we we want to add some description field? Or perhaps prefixing [CORE] and [COMMUNITY]...just thiking loud here, would love to hear your thoughts in this direction @geovanisouza92 @Widdershin

**Comment by nickbalestra** [comment]
PS: The idea is to then make northbrook compile the README.md from the communityFlavor.json so that It will be easy to keep the list up-todate by having people to simply PR against that file.
(we could use something like [this](https://github.com/IonicaBizau/json2md))

File: create-cycle-app/index.js [hunk 1]
```diff
@@ -70,34 +88,51 @@ function createApp (name, verbose, flavor) {
...
-  console.log(chalk.cyan('Fetching available flavors'))
+  console.log(chalk.cyan('Fetching community flavors...'))
   console.log()
 
-  // NOTE: Maybe change the method to discover flavors
+  // NOTE: change the method to discover flavors to the published communityFlavors.json
+  // following is just for testing
   request({
-    url: 'https://api.github.com/gists/0f33b55f62baca22c6bdb73b56333311',
+    url: 'https://api.github.com/repos/nickbalestra/create-cycle-app-community-flavors/contents/communityFlavors.json',
```
Comment (geovanisouza92) [inline create-cycle-app/index.js]: Should this point to your own repo? There is a similar file commited above.

Comment (nickbalestra) [inline create-cycle-app/index.js]: yup that was just for testing purpose, it should probably point to: `https://api.github.com/repos/cyclejs-community/create-cycle-app/contents/communityFlavors.json`

**Comment by geovanisouza92** [comment]
I like the changes, but not so sure about the split and the dependency on external files/repos to manage this. It's kinda against the monorepo idea.

**Comment by nickbalestra** [comment]
@geovanisouza92 : What you mean with `dependency on external files/repos`? Everything should be pretty much self contained no?

**Comment by geovanisouza92** [comment]
I choosed the wrong words, sorry. I mean, a separated repo for `*Flavors.json`, my bad, I didn't understood the diffs correctly.

Overall, it seems nice. Lets try this.

**Pull request merged**